ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -58,12 +70,15 @@ class SegmentPillButton: UIButton {
 
     private let item: SegmentItem
 
-    private let unreadDotLayer: CALayer = {
+    private func initUnreadDotLayer() -> CALayer {
```

Comment:
nit: should we be using "init" in this function name?

Reply:
That's what we named it in the Pill Button.

Reply:
I don't think we can use a closure initialized property because it refers to elements in `self`, so I think we need to go for a `lazy var` instead.

Reply:
@mischreiber lazy vars can't be computed properties.
I think you're referring to the fact that non-lazy vars can't refer to self in their initialization.

But this is what I'm suggesting:

```
    private lazy var unreadDotLayer: CALayer = {
        let unreadDotLayer = CALayer()
        let unreadDotSize = tokens.unreadDotSize
        unreadDotLayer.bounds.size = CGSize(width: unreadDotSize, height: unreadDotSize)
        unreadDotLayer.cornerRadius = unreadDotSize / 2
        return unreadDotLayer
    }()
```

Reply:
Ah yep that makes sense. Dang, this stuff confuses me.